crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs

```diff
@@ -154,14 +179,27 @@ impl<T: TransportConnect> SequencerAppender<T> {
             }
         };
 
-        if is_cancelled {
-            tracing::trace!("appender task cancelled");
-        } else {
-            tracing::trace!("appender task completed");
+        match final_state {
+            SequencerAppenderState::Done => {
+                tracing::trace!("appender task completed");
+            }
+            SequencerAppenderState::Cancelled => {
+                tracing::trace!("appender task cancelled");
+            }
+            SequencerAppenderState::Sealed => {
+                tracing::trace!("appender ended because of sealing");
```

Comment:
Should we add context to these log messages to be consistent with the rest? (loglet_id, etc.)

Reply:
Do we need to if those attributes already set on the span (via instrument) ?

Reply:
We don't.